lib/valkyrie/persistence/solr/persister.rb

```diff
@@ -16,7 +16,11 @@ def initialize(adapter:)
 
     # (see Valkyrie::Persistence::Memory::Persister#save)
```

Comment:
We should add documentation about return values. I guess maybe in `Valkyrie::Persistence::Memory::Persister#save`?